_extras/guide.md

```diff
@@ -245,13 +245,20 @@ stack_selection.columns = stack_selection.columns.droplevel()
 
 ### Challenge - Changing Types
 
+* Try converting the column `plot_id` to floats using `surveys_df.plot_id.astype("float")`. Next try converting `weight` to an integer. What goes wrong here? What is Pandas telling you?
+
 Pandas cannot convert types from float to int if the column contains NaN values.
 
 ### Challenge - Counting
+
+* Count the number of missing values per column. Hint: The method .count() gives you the number of non-NA observations per column. Try looking to the .isnull() method.
+
+~~~
 surveys_df.isnull()
+~~~
+{: .language-python}
 
-If the students have trouble generating the output, or anything happens with that, there is a file
-called "sample output" that contains the data file they should generate.
+If the students have trouble generating the output, or anything happens with that, there is a file called "sample output" that contains the data file they should generate.
```

Comment:
"there is a folder called `sample_output`, which contains the data file they should generate. It is not included with the data in the setup steps."

Most instructors don't have the learners clone the repo, as Git and GitHub are not part of the Data Carpentry core. So learners typically don't have this file, but the instructor could provide it.

Reply:
Nice call April. Perhaps this is a good wording?

```suggestion
If the students have trouble generating the output, or anything happens with that, there is a file called "sample output" that contains the data file they should generate. It is not included with the data in the setup steps, so remember to have it at hand and provide it if needed.
```

Reply:
@NPalopoli, note "folder" vs "file".
Another note: "there is" does not answer the question of "where". I suggest saying something like `This repository has a folder called...`

Reply:
Thank you @maxim-belkin once and again! Should be correct now.